pkgs/applications/science/biology/genmap/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "genmap";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "cpockrandt";
+    repo = "genmap";
+    rev = "genmap-v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-7sIKBRMNzyCrZ/c2nXkknb6a5YsXe6DRE2IFhp6AviY=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  # disable tests & benchmarks as these attempt to fetch googletest
+  # and googlebenchmark via git
+  preConfigure = ''
+    echo > tests/CMakeLists.txt
+    echo > benchmarks/CMakeLists.txt
+  '';
```

Comment:
A patch that uses the `gtest` package instead would be nice, benchmarks should indeed not be run inside Nixpkgs though.

Reply:
Patched, all tests passing.